packages/stripe_platform_interface/lib/stripe_platform_interface.dart

```diff
@@ -10,7 +10,7 @@ export 'src/models/create_token_data.dart';
 export 'src/models/errors.dart';
 export 'src/models/google_pay.dart';
 export 'src/models/payment_intents.dart';
-export 'src/models/payment_methods.dart';
+export 'src/models/payment_methods.dart' hide BillingDetailsConverter;
```

Comment:
I did this because adding it to another file would yield to circular dependencies